tasks/run-matlab-build/v0/task.json

```diff
@@ -0,0 +1,34 @@
+{
+    "": "https://raw.githubusercontent.com/Microsoft/azure-pipelines-task-lib/master/tasks.schema.json",
+    "id": "c61c6fcc-00f1-4aa0-8513-31aad3340512",
+    "name": "RunMATLABBuild",
+    "friendlyName": "Run MATLAB Build",
+    "description": "Run a build using the MATLAB build tool. Use this task to run the software-build tasks specified in a file named buildfile.m in the root of your repository. To use the run-build task, you need MATLAB R2022b or a later release.",
```

Comment:
Do you like to drop "software-build"?

Reply:
I am thinking maybe just "build tasks". What's you're opinion on that?

Reply:
Fine by me. We also can say "MATLAB build tasks".

@mcafaro, what do you think?

Reply:
Sounds good to me.

Reply:
updated